### PR TITLE
Bsi645

### PR DESCRIPTION
**Pull request by opetroulas**

### CODE REVIEW

File: uiengine/components/PickSelection/LinkedEventWinnerQuestion/LinkedEventWinnerQuestion.container.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+const mapState = ( store, { answerSheet, question } ) => {
+    if ( !question || !answerSheet ) return failureHelper()
+
+    const { choiceIds, questionData } = question;
+    const { linkedQuestionIds } = questionData;
+    if ( !linkedQuestionIds ) return failureHelper();
+
+    let myPicks = [];
```
Comment (Elimor5) [inline uiengine/components/PickSelection/LinkedEventWinnerQuestion/LinkedEventWinnerQuestion.container.js]: this could be  a `const`

Comment (opetroulas) [reply uiengine/components/PickSelection/LinkedEventWinnerQuestion/LinkedEventWinnerQuestion.container.js]: ¯\_(ツ)_/¯

File: uiengine/components/PickSelection/PickSelection.component.js [hunk 2]
```diff
@@ -92,7 +120,7 @@ export class PickSelectionView extends Component {
             }
 
             const { error, success } = result;
-
+            console.log("error = ", error );
```
Comment (Elimor5) [inline uiengine/components/PickSelection/PickSelection.component.js]: Is this  console log necessary?

Comment (opetroulas) [reply uiengine/components/PickSelection/PickSelection.component.js]: no but we're in testing so its useful sometimes. Eventually I clean it up but not pressing atm

File: uiengine/components/onboarding/SignInSignUp/SignInSignUp.component.js [hunk 3]
```diff
@@ -43,8 +43,11 @@ export class SignInSignUpView extends Component {
         if ( result.error ) {
             // ToDo real error handling
             this.props.showPopup({
+
                 title: "Error Logging In Error Code: " + result.error.errorCode + " Error Message: " + result.error.errorMessage + " Status Code: " + result.error.statusCode,
-                okButtonLabel: "Ok"
+                okButtonLabel: "Ok",
+              okButtonCallback: () => {},
```
Comment (Elimor5) [inline uiengine/components/onboarding/SignInSignUp/SignInSignUp.component.js]: Maybe add a todo here to keep track of what needs to be done in the callback?

File: uiengine/i18Files/pennNascar.json [hunk 4]
```diff
@@ -114,6 +114,11 @@
 
   "pickSelection": {
     "submitPicks": "submit picks",
+    "savePicks": "save picks",
```
Comment (Elimor5) [inline uiengine/i18Files/pennNascar.json]: why is `save  picks` all lower case?

File: uiengine/networkingResources/contests.js [hunk 5]
```diff
@@ -169,6 +169,7 @@ export class ContestsNI extends Networking {
             ]
         };
 
+        console.log("params = ", params );
```
Comment (Elimor5) [inline uiengine/networkingResources/contests.js]: is console log required?

File: uiengine/components/onboarding/SignInSignUp/SignInSignUp.component.js (hunk 3 above)
Comment (opetroulas) [reply uiengine/components/onboarding/SignInSignUp/SignInSignUp.component.js]: we actually want no real callback (ok button on this just dismisses the popup)

File: uiengine/i18Files/pennNascar.json (hunk 4 above)
Comment (opetroulas) [reply uiengine/i18Files/pennNascar.json]: client does to uppercase so doesn't really matter

**Pull request merged**